### DISCUSSION TITLE
Community Updates

### DISCUSSION THREAD

**Discussion by endigo9740** [skeletonlabs/skeleton#818] [Announcements]
Hey everyone, the moderation team and I have recently completed a round of updates to help improve the Skeleton community. The goal is to create a better and more inclusive community as the project continues to grow larger. We welcome your thoughts in the comment section below if you have suggestions or feedback about any of these changes.

- [Join our Discord Community](https://discord.gg/EXqV7W8MtY)

## Updated Code of Conduct

![Screen Shot 2023-01-12 at 3 45 42 PM](https://user-images.githubusercontent.com/1509726/212187906-c38de7e5-9891-4d43-a00b-ef7a3518ef7f.png)

We've updated our [Code of Conduct](https://github.com/skeletonlabs/community/blob/main/CODE_OF_CONDUCT.md) and will begin enforcing it for all Skeleton communities including GitHub and Discord. If you're curious about our enforcement policies, we cover these in the document itself. Note this is based on the same Code of Conduct the Svelte project utilizes.

## Discord Rule Screening

New members on Discord will now be met with a consent screen to ensure you conform with our rules. These are pulled directly from the Code of Conduct linked above and will be retroactive for existing users. Please keep these in mind.

## Discord Support

![Screen Shot 2023-01-12 at 3 48 44 PM](https://user-images.githubusercontent.com/1509726/212188387-4d5f2e1f-9b36-4b57-bbb4-a0645f7f8221.png)

We've introduced [a new stickied post on Discord](https://discord.com/channels/1003691521280856084/1062850506495635467) for the `#support` channel that covers our rules and guidelines for support messages. These are designed to help us help you. If your post does not follow these guidelines it may be removed by a moderator.

## Discord Bot

![Screen Shot 2023-01-12 at 3 52 43 PM](https://user-images.githubusercontent.com/1509726/212189068-98449d7a-093a-451c-9da6-17ca8a0bca54.png)

We've introduced a new Discord bot to help our moderators. This comes with many useful commands which can be found with `/help`, as well an ever growing set of custom commands (shown above). To list custom commands use `/tags list` - this will send a direct message listing all available commands. Type `!docs` for example to see a link to the documentation site. Note we'll continue to improve our tags and auto-moderation features over time.

## Discord Channel Rules

![Screen Shot 2023-01-12 at 3 47 24 PM](https://user-images.githubusercontent.com/1509726/212188168-0a828b86-fc24-4251-8015-d32b2b73dc64.png)

We've done a pass through each channel on Discord to update our rules. These are the small blurbs at the top of the channel section. Use these as a quick reference to ensure your posts are on topic!

**Comment by vdawg-git** [comment]
The Discord invite is invalid

**Reply by endigo9740** [reply]
Are you using this link?

https://discord.gg/EXqV7W8MtY

It's the same link that on the homepage of our website. Working fine for me a couple other members. Last person to join was an hour ago and would have had to use that link.

What I see:

![image](https://user-images.githubusercontent.com/1509726/213894142-58c81582-0bde-40a2-9f6a-50fc9603e089.png)

**Reply by vdawg-git** [reply]
<img width="517" alt="image" src="https://user-images.githubusercontent.com/28539403/213924761-80c707d5-f49f-407b-ad98-eb83586048e9.png">
Turns out Discord does not like VPNs. It worked when it was disabled.
